core/examples/luigi-example-openui5/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "luigi-example-openui5",
+  "private": true,
+  "version": "0.1.0",
+  "author": "SAP SE",
+  "description": "Luigi with OpenUI5 Demo App",
+  "scripts": {
+    "build": "ui5 build preload --all=true",
+    "start": "ui5 serve --open index.html",
+    "serve": "ws --compress -d dist",
+    "postinstall": "cp -r node_modules/@kyma-project/luigi-core webapp/libs/luigi-core && cp -r node_modules/@kyma-project/luigi-client webapp/libs/luigi-client"
```

Comment:
I would recommend this solution:
"postinstall": "cp -r node_modules/@kyma-project/luigi-core webapp/libs/ && cp -r node_modules/@kyma-project/luigi-client webapp/libs/"

When you run `npm install` again, luigi-core and luigi-client will copied in libs/luigi-core/luigi-core (the same for luigi-client)

Reply:
I see, I am running `npm install` multiple times but I was not getting that behaviour. I suppose it behaves differently in different environments. (since Zarko also said the app suddenly stopped working)